RELEASES.md

```diff
@@ -1,3 +1,109 @@
+Version 1.41.0 (2020-01-30)
+===========================
+
+Language
+--------
+
+- [You can now pass type parameters to foreign items when implementing
+  traits.][65879] E.g. You can now write `impl<T> From<Foo> for Vec<T> {}`.
+- [You can now use any receiver in the `self` position.][64325] E.g. you can
+  now write `fn foo(self: Box<Box<Self>>) {}`. Previously only `Self`, `&Self`,
+  `&mut Self`, `Arc<Self>`, `Rc<Self>`, and `Box<Self>` were allowed.
+- [You can now use any valid identifier in a `format_args` macro.][66847]
+  Previously this was a strict subset.
+- [You can now use `!`/"Never type" in any type position.][65355]
+
+Compiler
+--------
+
+- [Rustc's parser now allows visibility in trait items and enum variants.][66183]
+  These are still rejected when compiled, however they can be seen and parsed
+  by procedural macros.
+- [Rustc will now warn if you have unused `'loop` labels.][66325]
+- [Removed support for the `i686-unknown-dragonfly` target.][67255]
+- [Added tier 3 support\* for the `riscv64gc-unknown-linux-gnu` target.][66661]
+- [You can now pass an arguments file passing the `@path` syntax
+  to rustc.][66172]
```

Comment:
```suggestion
  to rustc.][66172]
    Note that the format differs somewhat from what is found in other tooling; see [docs](https://doc.rust-lang.org/nightly/rustc/command-line-arguments.html#path-load-command-line-flags-from-a-path) for more information.
```

(Basically I would like there to be a link to the docs here, but I don't care too much how we accomplish that).